docs/tools/riley-swf-patcher.md

```diff
@@ -0,0 +1,16 @@
+#Riley's SWF Patcher
+
+Riley's SWF patcher is a tool that you can use to inject ActionScript code into existing Flash games.
+
+## Download
+
+- [Download the tool here](https://github.com/rayyaw/flash-patcher)
+
+## Usage
+
+To use the tool, download `apply-patch.py`. Once you've done this, you will need to edit the `JPEXS_PATH` and `JPEXS_ARGS` variables to make the patcher run on your system.
```

Comment:
Avoid explaining usage on this site when you can just link to the documentation. We only give usage instructions if there are none already in existence, or if the software has been abandoned (e.g. Swf2Exe), not for fresh software. You wouldn't want to have to come back here and update it if you changed something over there :P

Reply:
Removed the documentation, it just has a direct link to the patcher README now